packages/pyxis-react/src/components/Modal/Footer/Footer.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC, ReactNode } from 'react';
+import classNames from 'classnames';
+
+const getClasses = (isSticky?: boolean, alt?: boolean): string => classNames('modal__footer', {
+  'modal__footer--alt': alt,
+  'modal__footer--sticky': isSticky,
+});
+
+const getButtonClasses = (hasFullWidthButtons?: boolean) => classNames('modal__footer__buttons', {
+  'modal__footer__buttons--full-width': hasFullWidthButtons,
+});
+
+const Footer: FC<FooterProps> = ({
+  alt,
+  buttons,
+  children,
+  hasFullWidthButtons,
+  id,
+  isSticky,
+  text,
```

Comment:
When possible (especially for booleans) we put default values, so we can avoid `undefined` in the functions that generate the classes

```suggestion
  alt = false,
  buttons,
  children,
  hasFullWidthButtons = false,
  id,
  isSticky = false,
  text,
```